lib/model/model.go

```diff
@@ -505,7 +533,11 @@ func (m *model) restartFolder(from, to config.FolderConfiguration, cacheIgnoredF
 			// Create a new fset. Might take a while and we do it under
 			// locking, but it's unsafe to create fset:s concurrently so
 			// that's the price we pay.
-			fset = db.NewFileSet(folder, to.Filesystem(), m.db)
+			var err error
+			fset, err = db.NewFileSet(folder, to.Filesystem(), m.db)
+			if err != nil {
+				return fmt.Errorf("restarting folder: %w", err)
```

Comment:
```suggestion
				return fmt.Errorf("restarting %s: %w", to.Decription(), err)
```

It may not be consistent, but I think some / many / all of these errors end up in m.fatal() and don't explicitly get the affected folder name added as context before printing / crashing / etc... So maybe we should always add the description to the error, worst case we get double the folder name in the final output but 🤷